dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/preheat/PreheatServiceTest.java

```diff
@@ -679,13 +508,9 @@ public void testUserPreheatCollection()
         params.setObjects( metadata );
 
         Preheat preheat = preheatService.preheat( params );
-        assertNotNull( preheat.getMap().get( PreheatIdentifier.CODE ) );
-        assertNotNull( preheat.getMap().get( PreheatIdentifier.UID ) );
 
-        assertFalse( preheat.getMap().get( PreheatIdentifier.CODE ).isEmpty() );
+        assertNotNull( preheat.getMap().get( PreheatIdentifier.UID ) );
```

Comment:
Any reason `preheat.getMap().get( PreheatIdentifier.UID )` isn't extracted to a local var so it can be reused in subsequent asserts?